initial-checks.sh

```diff
@@ -11,9 +11,21 @@ uname -s | grep -i linux
 check_for_failure "Please run this script on Linux"
 
 if [ "$EUID" -ne 0 ];then
-  check_for_failure "Please run as root or with sudo"
+  echo "Please run as root or with sudo"
+  exit 1
```

Comment:
@frosario When I tried running this script without sudo it didn't throw the error. It seems like check_for_failure requires a previous command to fail?